src/main/java/com/hotline/webdriver/WebDriverFacade.java

```diff
@@ -0,0 +1,29 @@
+package com.hotline.webdriver;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+public class WebDriverFacade {
+    private final WebDriver baseDriver;
+
+    public WebDriverFacade(WebDriverFactory webDriverFactory) {
+        this.baseDriver = webDriverFactory.getChromeDriver();
+    }
+
+    public WebDriver getBaseDriver() {
+        return baseDriver;
```

Comment:
What if `baseDriver` is not defined? There is a possibility of NullPointerException here